ufs/ccpp/data/MED_data.meta

```diff
@@ -0,0 +1,85 @@
+[ccpp-table-properties]
+  name = physics_type
+  type = ddt
+  dependencies = MED_typedefs.F90
+
+[ccpp-arg-table]
+  name = physics_type
+  type = ddt 
+[Init]
+  standard_name = MED_init_type_instance
+  long_name = instance of derived type MED_init_type
+  units = DDT
+  dimensions = ()
+  type = MED_init_type
+[Statein]
+  standard_name = MED_statein_type_instance
+  long_name = instance of derived type MED_statein_type
+  units = DDT
+  dimensions = ()
+  type = MED_statein_type
+[Interstitial]
+  standard_name = MED_interstitial_type_instance
+  long_name = instance of derived type MED_interstitial_type
+  units = DDT
+  dimensions = ()
+  type = MED_interstitial_type
+[Model]
+  standard_name = MED_control_type_instance
+  long_name = instance of derived type MED_control_type
+  units = DDT
+  dimensions = ()
+  type = MED_control_type
+[Coupling]
+  standard_name = MED_coupling_type_instance
+  long_name = instance of derived type MED_coupling_type
+  units = DDT
+  dimensions = ()
+  type = MED_coupling_type
+[Grid]
+  standard_name = MED_grid_type_instance
+  long_name = instance of derived type MED_grid_type
+  units = DDT
+  dimensions = ()
+  type = MED_grid_type
+[Sfcprop]
+  standard_name = MED_sfcprop_type_instance
+  long_name = instance of derived type MED_sfcprop_type
+  units = DDT
+  dimensions = ()
+  type = MED_sfcprop_type
+[Diag]
+  standard_name = MED_diag_type_instance
+  long_name = fields targeted for diagnostic output
+  units = DDT
+  dimensions = ()
+  type = MED_diag_type
+
+########################################################################
+[ccpp-table-properties]
+  name = MED_data
+  type = module
+  dependencies = MED_typedefs.F90
+  dependencies = ../../../../../FV3/ccpp/framework/src/ccpp_api.F90
```

Comment:
Wondering if there is a better way to specify the relative path to this file.

Reply:
@climbfuji yes. if there is a way to specify the path in a more elegant way, that would be great. Is there any macro (like the variables in the CCPP configure file) in CCPP side that could be used in meta file?

Reply:
@climbfuji I seems that `ccpp_api.F90` is removed with your recent PR (`https://github.com/NOAA-EMC/fv3atm/commit/bffbeeb523bba6119dd17abfc73ab9125a4f5e07`) and i was getting error to compile CCPP host model under CMEPS. Anyway, I could fix it by removing the part that points `ccpp_api.F90` in the meta file and also replacing `use ccpp_api, only: ccpp_t` with `use ccpp_types,  only: ccpp_t` in the code.